terraform/outputs.tf

```
@@ -0,0 +1,9 @@
+output "iam_role_name" {
+  description = "The name of the IAM Role"
+  value       = aws_iam_role.doit-management-role.name
```

@tibor-szalai Don't forget to change the outputs. ;)
```suggestion
  value       = aws_iam_role.doit_management_role.name
```

---

I did forget. Thanks for reminding me. :)